napari_plugin_engine/dist.py

```diff
@@ -139,7 +139,9 @@ def standard_metadata(plugin: Any) -> Dict[str, Optional[str]]:
     if not get_dist(plugin):
         _top_level_module_to_dist.cache_clear()
         if not get_dist(plugin):
-            raise ValueError(f"could not find metadata for {plugin}")
+            return {}
+            # TODO: decide appropriate behavior here.
+            # raise ValueError(f"could not find metadata for {plugin}")
```

Comment:
What's the story here?

Reply:
nvmd, just read above! 😅

Reply:
yeah, that's the thing I "broke" ... made it an exception to ask for metadata from an uninstalled package.  just decided to kick that can down the road and let it pass for now.